src/integrationTest/resources/application-integration.yaml

```diff
@@ -61,5 +61,5 @@ config:
   allowedCaseTypes: ${ALLOWED_CASE_TYPES:asylum,wacasetype,sscs,civil,generalapplication,CARE_SUPERVISION_EPO,PRLAPPS,ET_EnglandWales,ET_EnglandWales_Listings,ET_EnglandWales_Multiple,ET_Scotland,ET_Scotland_Listings,ET_Scotland_Multiple,ET_Admin,privatelaw_exceptionrecord,benefit,CriminalInjuriesCompensation}
   taskMandatoryFieldCheckEnabled: ${MANDATORY_TASK_FIELD_CHECK_ENABLED:true}
   taskMandatoryFields: ${MANDATORY_TASK_FIELDS:taskName,taskId,taskType,dueDateTime,state,securityClassification,title,majorPriority,minorPriority,executionTypeCode,caseId,caseTypeId,caseCategory,caseName,jurisdiction,region,location,created,roleCategory,workTypeResource}
-  taskMandatoryFieldsProvidedByClient: ${MANDATORY_TASK_FIELDS_PROVIDED_BY_CLIENT:taskName,taskType,caseTypeId,jurisdiction} //RWA-3970
+  taskMandatoryFieldsProvidedByClient: ${MANDATORY_TASK_FIELDS_PROVIDED_BY_CLIENT:name,taskType,caseId}
```

Comment:
Maybe rename this field to be something like `initiationRequestRequiredFields`?  To distinguish it from the other mandatory fields as this is specifically about the request body for initiations.

Reply:
updated